### ISSUE TITLE
compute the lane_width

### ISSUE THREAD

**Issue by pocky111** [hirotomusiker/CLRerNet#20]
Hello, I have been reading your code recently, and I don't quite understand why lane_width= 15/800 in the calculation of loss function. Could you please explain it to me?

**Comment by hirotomusiker** [comment]
`lane_width` is the half-width to form virtual lane markers and calculate IoU. In LineIoU ([CLRNet](https://github.com/Turoad/CLRNet/blob/main/clrnet/models/losses/lineiou_loss.py#L4)), it's called `length` and set to 15 in the image resolution of 800. So we set it to 15/800 in the relative coordinate system (0 to 1.0) as the default value.
We actually set lane_width to 7.5 / 800 to better represent the [lane width for evaluation](https://github.com/hirotomusiker/CLRerNet/blob/main/libs/datasets/metrics/culane_metric.py#L22-L34) for CLRerNet. We will include the settings in our training codebase. Thank you!
